src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -532,6 +536,9 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         auto pfnScrollPositionChanged = std::bind(&TermControl::_TerminalScrollPositionChanged, this, std::placeholders::_1, std::placeholders::_2, std::placeholders::_3);
         _terminal->SetScrollPositionChangedCallback(pfnScrollPositionChanged);
 
+        _autoScrollTimer.Interval(std::chrono::microseconds(static_cast<int>(1.0 / 30.0 * 1000000)));

```

Comment:
Can we just make this time delta a named constant somewhere?

Reply:
Line above or as member? (Btw. I've seen the oppsite request on some other PR - to delete the named constant when it's only used once.)